core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -764,12 +815,70 @@ def __do_switch_wal(self):
         Let PostgreSQL write latest transaction into a new WAL file now
         """
         try:
-            self.db_con.run("select pg_switch_wal()")
+            lsn = self.db_con.run("select pg_switch_wal()")[0][0]
```

Comment:
Maybe we should add to the doc string that this function returns the lsn of the (last ? current ?) wal file.

Reply:
Good idea